packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/AssetType.java

```diff
@@ -12,5 +12,6 @@ public enum AssetType {
 	WORKFLOW,
 	ARTIFACT,
 	CODE,
-	DOCUMENT
+	DOCUMENT,
+	NETCDF
```

Comment:
Why do we have this here?

Reply:
You should be seeing that change - that was a merge error but I am not showing that as current in the PR